src/Plugin/Field/FieldWidget/GeocoderAutocomplete.php

```diff
@@ -1,8 +1,4 @@
 <?php
-/**
- * @file
- * Contains \Drupal\geocoder_autocomplete\Plugin\Field\FieldWidget\GeocoderAutocomplete.
- */
```

Comment:
Do you use the latest version of phpcs drupal standard?
I use Verison 8.x-2.7 from https://www.drupal.org/project/coder